Proposition de tradustion française et correction de la vue "Ponto"

pasgou opened the pull request:
Traduction française des modules
Correction de la vue des services : pour Ponto, c'était qonto qui était affiché.

pasgou committed:
French Translation

pasgou committed:
Update online_bank_statement_provider.xml

pedrobaeza commented:
Why this is not done on Weblate? https://odoo-community.org/page/translate

pasgou commented:
C’est une erreur de ma part.
Je travaille avec un outil de traduction qui me permet de travailler plus vite ...
Je n’ai pas transféré la traduction que j’ai faite sur webtranslate alors que j’y suis inscrit. Pardonnez moi.



> Le 17 févr. 2021 à 11:58, Pedro M. Baeza <notifications@github.com> a écrit :
> 
> 
> Why this is not done on Weblate? https://odoo-community.org/page/translate <https://odoo-community.org/page/translate>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/OCA/bank-statement-import/pull/372#issuecomment-780476410>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ACFZ54P44ONEZ2YRAS7NL3DS7OONZANCNFSM4WI2PVZQ>.
>

pasgou committed:
traduction mise à jour

ypapouin commented:
If you are registered on Weblate, you can use PO download/upload to edit an external tool.
Note that I had to request access rights from an admin to get an access to these commands.

![image](https://user-images.githubusercontent.com/22446243/109663317-42109700-7b6c-11eb-97a5-b740d0a8c0fb.png)

pasgou commented:
Oui
Mais là j’avais oublié de le faire
Et je l’utilise d’habitude
Peu car peu de temps disponible mais je le fais.
J’ai fais un peu vite cette correction en même temps que celle concernant
le code lui même.

Le mar. 2 mars 2021 à 15:34, Yann Papouin <notifications@github.com> a
écrit :

> If you are registered on Weblate, you can use PO download/upload to edit
> an external tool.
> Note that I had to request access rights from an admin to get an access to
> these commands.
>
> [image: image]
> <https://user-images.githubusercontent.com/22446243/109663317-42109700-7b6c-11eb-97a5-b740d0a8c0fb.png>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OCA/bank-statement-import/pull/372#issuecomment-788951814>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACFZ54PW7T5BIDFKRXM6PR3TBTZOPANCNFSM4WI2PVZQ>
> .
>
-- 
Cordialement

Pascal GOUHIER


COPAGEX
Société d'expertise comptable
Membre de la Communauté des Experts en Gestion
8 Boulevard Albert Einstein
Parc de la Rivière
44300 NANTES

T : 02 40 58 56 49
F : 02 40 95 11 81

http://www.copagex.fr

pasgou committed:
Update Translation

pasgou committed:
Update online_bank_statement_provider_ponto.py

Add a function to collect account ending balance.
To test, Add a new synchronisation function. This one could be adapted in order to have only one synchronisation function with a test on 'subtype' parameter that adapt the url to the context.

pasgou committed:
Revert "Update online_bank_statement_provider_ponto.py"

This reverts commit b28ef5137056103ef4b2b84792d230440e38583d.
